### PR TITLE
Added Documentation Folder

### PR DESCRIPTION
**Pull request by mrzengel**
Note: Right now, the Github Pages points to the docs folder on the documentation branch. When the PR is approved, I'll point it to main.

Second Note: Github action has a built in Github action that looks for changes in the documentation directory

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ This project integrates [Zenodo](https://zenodo.org) into Jupyter Lab extension.
 # Requirements
 JupyterLab > 4, < 5
```
Comment (garciagenrique) [inline README.md:10]: Small decorative detail: can you please add each requirement as a bullet list, and put the eOSSR lib as a link pointing to the git repo, please

Comment (garciagenrique) [inline README.md:10]: Small decorative detail: can you please add each requirement as a bullet list, and put the eOSSR lib as a link pointing to the git repo, please

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# Zenodo Jupyterlab Extension
+[![Build and install extension](https://github.com/vre-hub/zenodo-jupyterlab-extension/actions/workflows/build.yaml/badge.svg)](https://github.com/vre-hub/zenodo-jupyterlab-extension/actions/workflows/build.yaml)
+
+[Zenodo JupyterLab extension repository](https://github.com/vre-hub/zenodo-jupyterlab-extension)\
+**This project is currently in development and is subject to change.**
+
+This project integrates [Zenodo](https://zenodo.org) into Jupyter Lab extension.
+
+# Requirements
+JupyterLab > 4, < 5\
```
Comment (garciagenrique) [inline docs/index.md:15]: Same detail here, thanks !

**Review by garciagenrique — APPROVED** [review]
Left two small comments. I will read now the docs from the formatted site and comment if I find anything weird, but in general everything looks very nice to me!

File: docs/index.md (hunk 2 above)
Comment (mrzengel) [reply docs/index.md]: These should all be updated now

**Comment by garciagenrique** [comment]
I think that after having moved the documentation, we can close this PR @mrzengel. What do you think ? 

Can you also make a small PR with the changes on the README ?

**Comment by garciagenrique** [comment]
Solved via https://github.com/vre-hub/vre-hub.github.io/pull/8 and https://github.com/vre-hub/vre-hub.github.io/pull/10

**Pull request closed without merging**